.github/workflows/tests.yaml

```diff
@@ -1,17 +1,21 @@
-name: Run pytest tests
+name: Tests
 
 on:
   push:
     branches-ignore:
-      - github-pages/*
+      - 'main'
```

Comment:
why not on main?

Reply:
Good point, this is a leftover. I wanted to refactor more but reverted it then and decided to address it later.